Assets/BossRoom/Scripts/Server/Game/State/ServerPostGameState.cs

```diff
@@ -23,15 +23,10 @@ public override void NetworkStart()
             else
             {
                 bool won = (bool)GameStateRelay.GetRelayObject();
-                StartCoroutine(CoroSetWinLoss(won));
-            }
-        }
 
-        private IEnumerator CoroSetWinLoss(bool won)
-        {
-            yield return new WaitForSeconds(1);
-            GetComponent<PostGameData>().GameBannerState.Value =
-                (byte)(won ? PostGameData.BannerState.Won : PostGameData.BannerState.Lost);
+                GetComponent<PostGameData>().GameBannerState.Value =
```

Comment:
Can we serialize the `PostGameData` component and cache it?

Reply:
you got it.